src/structs/atom.rs

```diff
@@ -473,3 +473,4 @@ impl PartialEq for Atom {
             && self.b_factor == other.b_factor
     }
 }
+impl Eq for Atom {}
```

Comment:
As Atom contains multiple f64 values it cannot implement Eq as f64 does not implement Eq. [See this question](https://users.rust-lang.org/t/what-is-the-difference-between-eq-and-partialeq/15751)